.github/workflows/test-and-deploy.yml

```diff
@@ -1,15 +1,37 @@
-name: Release
+name: Test and Deploy
```

Comment:
Also need to update the badge in the README.

Reply:
Thanks. Updated the README with the new status badge.